OWNERS_ALIASES

```diff
@@ -66,14 +65,13 @@ aliases:
     - reedcort
     - mrWinston
   srep-functional-leads:
-    - mrbarge
     - rafael-azevedo
     - iamkirkbater
     - bng0y
     - tonytheleg
```

Comment:
Should add `ravitri` to the `srep-funtional-leads`?

Reply:
Thanks Wenqi! This change will need to be part of boilerplate - https://github.com/openshift/boilerplate/blob/master/boilerplate/openshift/golang-osd-operator/OWNERS_ALIASES#L67 

I'll raise a PR for that change. Thanks to notice it.